feature-utils/poly-analysis/.gitignore

```diff
@@ -0,0 +1,13 @@
+## editors
+/.idea
+/.vscode
+
+## system files
+.DS_Store
+
+## npm
+/node_modules/
+/npm-debug.log
+
+# build
+/dist/
```

Comment:
I think this might be covered already by the upper-level `.gitignore`, isn't it?

Reply:
... and it's not covered. It should probably be. But we're good for the time being.